scripts/standardising.py

```diff
@@ -12,61 +12,63 @@
 arguments = parser.parse_args()
 source = arguments.source
 destination = arguments.destination
-# TODO if destination needs write permission we have to handle this
 
 get_log().info("standardising", source=source, destination=destination)
-
-src_bucket_name, src_file_path = parse_path(source)
 dst_bucket_name, dst_path = parse_path(destination)
-get_log().debug("source", bucket=src_bucket_name, file_path=src_file_path)
 get_log().debug("destination", bucket=dst_bucket_name, file_path=dst_path)
 dst_bucket = get_bucket(dst_bucket_name)
 
+# TODO Include TDE-411 here
+file_list = [source]
+
 with tempfile.TemporaryDirectory() as tmp_dir:
```

Comment:
the temp folder should be deleted between files, otherwise you might fill up the disk